client/components/ConfigureActiveRuns/index.jsx

```diff
@@ -393,6 +395,9 @@ class ConfigureActiveRuns extends Component {
                                                     deleteTechnologyRow={
                                                         this.deleteTechnologyRow
                                                     }
+                                                    editingRow={
+                                                        this.state.editingRow
```

Comment:
Actually it's not quite this simple -- the purpose of having some rows look uneditable is because those rows are part of the previous configuration. We want the user to be able to tell which rows they added and which already exist up until the point they save.

For the code, this means the rows that come directly from the "activeRunConfiguration" global state should always look uneditable, though they can be deleted. Any new rows a user adds (before they click save) should remain editable until they click "save".

One way to implement this is to add a new flag called "editable" to all of the objects in `this.state.runTechnologyRows`. This attribute would set to false for all the at/browser pairs returned by  `getDefaultsTechCombinations`, and set to true for all new rows added (see the "push" in `handleTechnologyRowChange`). Then you can pass that value in here (`runTech.editable`).

Reply:
Good catch! I didn't see that the uneditable rows become editable. I'm on it!